server/src/controllers/server.controller.ts

```diff
@@ -95,6 +95,18 @@ export class ServerController {
 
   @Get('license')
   @Authenticated({ admin: true })
+  @ApiOkResponse({
+    schema: {
+      oneOf: [
+        {
+          $ref: getSchemaPath(LicenseResponseDto),
+        },
+        {
+          type: 'null',
+        },
+      ],
+    },
+  })
```

Comment:
I understand this works, but idk why we even return null in the first place. I'm fine with just throwing an error if you are asking for a resource that doesn't exist. You can always check the about response to know if it is licensed in the just place. We have no other endpoints that return a dto or null and I see no reason to start doing that here. 

(I understand you are just making the typing correct)

Reply:
Good point. I've updated it to return a 404 error for non-existing licenses, consistent with the user license endpoint.